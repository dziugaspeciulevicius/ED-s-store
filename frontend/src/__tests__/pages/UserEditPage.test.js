
import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupTests";
import UserEditPage from "../../pages/UserEditPage.component"
jest.mock("../../pages/UserEditPage.component");

describe("<UserEditPage />", () => {
    it("Renders <UserEditPage /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<UserEditPage />)).toMatchSnapshot();
    });
});