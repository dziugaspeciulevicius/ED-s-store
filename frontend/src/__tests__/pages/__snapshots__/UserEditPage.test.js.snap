// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserEditPage /> Renders <UserEditPage /> component correctly 1`] = `ShallowWrapper {}`;
